tests/always/test_providers_manager.py

```diff
@@ -384,6 +384,11 @@ def test_notification(self):
         notification_class_names = list(provider_manager.notification)
         assert len(notification_class_names) > 5
 
+    def test_auth_managers(self):
+        provider_manager = ProvidersManager()
+        notification_class_names = list(provider_manager.auth_managers)
+        assert len(notification_class_names) > 0
```

Comment:
```suggestion
        auth_manager_class_names = list(provider_manager.auth_managers)
        assert len(auth_manager_class_names) > 0
```
nit: variable name is a bit misleading here

Reply:
Indeed. Committed it straight away